README.md

```diff
@@ -95,6 +95,64 @@ async function main() {
     console.log('Results: ', { orderId, order, trades, orderCancellationSigningResult, cancellationResult })
 }
 ```
+
+### OrderBookApi
+
+`OrderBookApi` - is a main tool for working with [CoW Protocol API](https://api.cow.fi/docs/#/).
+Since the API supports different networks and environments, there are some options to configure it.
+
+#### Environment configuration
+```typescript
+import { OrderBookApi } from '@cowprotocol/cow-sdk'
+
+const orderBookApi = new OrderBookApi({
+  chainId: SupportedChainId.GOERLI,
+  env: 'staging' // <-----
+})
+```
+
+#### API urls configuration
+```typescript
+import { OrderBookApi } from '@cowprotocol/cow-sdk'
+
+const orderBookApi = new OrderBookApi({
+  chainId: SupportedChainId.GOERLI,
+  baseUrls: { // <-----
+    [SupportedChainId.MAINNET]: 'https://YOUR_ENDPOINT/mainnet',
+    [SupportedChainId.GNOSIS_CHAIN]: 'https://YOUR_ENDPOINT/xdai',
+    [SupportedChainId.GOERLI]: 'https://YOUR_ENDPOINT/goerli',
```

Comment:
Again, why would I want to change the url?

Reply:
Added more info